docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1329,26 +1334,29 @@ function getRemoteMediaSettings() {
   if (meeting && meeting.mediaProperties.remoteVideoTrack) {
     const videoSettings = meeting.mediaProperties.remoteVideoTrack.getSettings();
     const {frameRate, height} = videoSettings;
-
     remoteVideoResElm.innerText = `${height}p ${Math.round(frameRate)}fps`;
   }
 }
 
-let resolutionInterval;
+let localVideoResolutionInterval;
+let remoteVideoResolutionInterval;
 const INTERVAL_TIME = 3000;
 
-function meetingsResolutionCheckInterval() {
-  resolutionInterval = setInterval(() => {
+function localVideoResolutionCheckInterval() {
+  localVideoResolutionInterval = setInterval(() => {
     getLocalMediaSettings();
-    getRemoteMediaSettings();
   }, INTERVAL_TIME);
 }
 
-function clearMeetingsResolutionCheckInterval() {
-  localVideoResElm.innerText = '';
-  remoteVideoResElm.innerText = '';
+function remoteVideoResolutionCheckInterval() {
+  remoteVideoResolutionInterval = setInterval(() => {
```

Comment:
remoteVideoResolutionCheckInterval() is called only once per meeting, so doesn't necessarily need the same logic to clear the previous interval like I commented on localVideoResolutionCheckInterval(), but maybe it would be good to add it here anyway for completeness?

Reply:
Done